### DISCUSSION TITLE
View logs in kibana

### DISCUSSION THREAD

**Discussion by pdaniel** [dcm4che/dcm4chee-arc-light#2920] [Q&A] [answered]
I have configured dcm4cee to run using elasticsearch, logstash and kibana on a single host. I don't know. and i couldn't find in the documentation how to view the logs from elasticsearch. Is there a documentation on how to define the indexes in kibana or how to view application logs?

**Comment by vrindanayak** [comment, marked as answer]
@pdaniel : Added a wiki page for [View Audit Messages in Kibana](https://github.com/dcm4che/dcm4chee-arc-light/wiki/View-Audit-Messages-in-Kibana)
